riff-raff.yaml

```diff
@@ -45,7 +46,8 @@ deployments:
   frontend-static:
     type: aws-s3
     parameters:
-      bucket: aws-frontend-static
+      bucketSsmKey: /account/services/dotcom-static
```

Comment:
```suggestion
      bucketSsmKey: /account/services/dotcom-static.bucket
```

Reply:
Unfortunately the `aws-s3` type doesn't support `bucketSsmKey` yet. It's [in progress](https://github.com/guardian/riff-raff/pull/893) though!

Reply:
thanks @akash1810! I've restored the hard-coded bucket name now. Does this look good to you @ioannakok? Did we think we should do a CODE deployment first?

Reply:
Yep that's good @bryophyta 👍 It should be fine but let's do a CODE deployment first!